packages/pds/src/pipethrough.ts

```diff
@@ -89,17 +102,18 @@ export const formatUrlAndAud = async (
   if (!ctx.cfg.service.devMode && !isSafeUrl(url)) {
     throw new InvalidRequestError(`Invalid service url: ${url.toString()}`)
   }
-  return { url, aud }
+  return { url, aud, nsid }
 }
 
 export const formatHeaders = async (
   ctx: AppContext,
   req: express.Request,
   aud: string,
+  lxm: string,
   requester: string | null,
```

Comment:
Not necessarily a change we need to make now, but some of these functions have enough positional `string` args next to each other that it may be worth moving them to an object to avoid mixups.